daemon/archive.go

```diff
@@ -426,9 +432,6 @@ func (daemon *Daemon) containerCopy(container *container.Container, resource str
 		d, f := driver.Split(basePath)
 		basePath = d
 		filter = []string{f}
-	} else {
-		filter = []string{driver.Base(basePath)}
```

Comment:
No filter is needed in this case?

Reply:
This was to keep it symmetrical and I believe is used by older docker CLIs (this codepath is dead code for newer APIs, but not older ones). @cpuguy83 confirmed as well that cp works as intended for an older docker CLI (like 1.8).

Reply:
Correct, because no filter means everything.